_posts/2022-09-07-#25957.md

```diff
@@ -10,14 +10,26 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- This PR is a re-attempt of [PR 15845](https://github.com/bitcoin/bitcoin/pull/15845)
+from 2019, which was closed without being merged. PR 15845 was the
+subject of an earlier [review club](https://bitcoincore.reviews/15845).
+- Please see the notes for that review club meeting, as they apply here too.
+
 ## Questions
-1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)?
--->
+1. Did you review the PR?
+[Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)?
+1. Why would a node operator enable BIP 158 filters (`-blockfilterindex=1`)? Does this PR affect the motivation?
+1. What downsides, if any, are there to enabling BIP 158 filters?
+1. What is the advantage of descriptor wallets compared to legacy wallets, especially w.r.t. the creation of the filter set?
+(Hint: what exact type of data do we need to put into the filter set?)
+1. On a new descriptor wallet with default settings (i.e. 'keypoolsize=1000'), how many elements would we need to put the filter set?
+(Hint: how many descriptors are created by default? can be simply tried out via the listdescriptors RPC)
+1. What is the difference between active and non-active descriptors (and `DescriptorScriptPubKeyMans`, respectively)
+and why does this distinction matter for this PR?
+1. What was the flaw of an earlier version of this PR and how do we solve that?
```

Comment:
Would refer to 15845 specifically if that's the one you mean.

Reply:
The notes should contain some information about why the older PR was abandoned, especially if it's a question. Alternatively, just link to the discussion as a hint for this question.